index.html

```diff
@@ -1,67 +1,99 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>판다마켓</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <header>
-        <div class="header">
-            <a href="/" >
-                <img src="images/ic_pandamarket_logo.png" alt="판다마켓 로고" class="logo"></a>
-            <a href="/login">
-                <img src="images/ic_login.png" alt="로그인" class="login"></a>
-        </div>
-    </header>
+<html lang="ko">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>판다마켓</title>
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body>
+        <header>
+            <div class="header">
+                <a href="/"> <img src="images/logo.png" alt="판다마켓 로고" class="logo" /></a>
+                <a href="/login.html"> <img src="images/ic_login.png" alt="로그인" class="login" /></a>
+            </div>
+        </header>
         <section class="headline-wrap">
             <div class="headline">
                 <div class="headline-content">
                     <h1 class="headline-heading">
-                        일상의 모든 물건을<br>
+                        일상의 모든 물건을<br />
                         거래해 보세요
                     </h1>
-                    <a href="/items">
-                    <h2 class="headline-button" alt="구경하러가기">구경하러 가기</h2></a>   
+                    <a href="/items"> <h2 class="headline-button" alt="구경하러가기">구경하러 가기</h2></a>
```

Comment:
❗️ 수정요청

지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

+h2 태그에는 alt 속성이 없습니다. a 태그에 classname을 통해 스타일링 하시는 것을 추천드려요!

```suggestion
                    <a href="/items" class="headline-button">구경하러 가기</a>
```